infrastructure/templates/infrastructure/powerplant_detail.html

```diff
@@ -121,9 +123,20 @@ <h3>Estimated Output</h3>
       <div class="row double">
           <section>
               <h3>Fuel</h3>
-              {% for fuel in object.fuels.all %}
-                <div>{{ fuel }}</div>
-              {% endfor %}
+              <div class="row double">
```

Comment:
Since these changes are unrelated to the fuel changes, ideally this branch would have come off of ``mou-backend`` directly.

Reply:
I think did that because `docker-compose` wasn't on the mou-backend. Should I rebase it?